cmd/controller/main.go

```diff
@@ -331,8 +338,16 @@ func main() {
 	}
 
 	for _, registry := range strings.Split(nodeInsecureRegistries, ",") {
-		if strings.TrimSpace(registry) != "" {
-			runOptions.node.InsecureRegistries = append(runOptions.node.InsecureRegistries)
+		trimmedRegistry := strings.TrimSpace(registry)
```

Comment:
Nit: Mind using here and below the shorted ` if trimmedRegistry := strings.TrimSpace(registry); trimmgedRegeistry != "" {`

Reply:
Will change.